test/unit/tracks/text-track.test.js

```diff
@@ -265,56 +262,60 @@ test('tracks are parsed if vttjs is loaded', function() {
 
   // use proxyquire to stub xhr module because IE8s XDomainRequest usage
   let xhrHandler;
-  let TextTrack = proxyquire('../../../src/js/tracks/text-track.js', {
+  const TextTrack_ = proxyquire('../../../src/js/tracks/text-track.js', {
```

Comment:
since TextTrack is imported at the top eslint complains about redefining a variable